src/components/slider/slider.material.scss

```diff
@@ -187,7 +183,6 @@ $thumb-label-height: rem(30px);
     color: contrast-color(gray, 700);
     background: color(gray, 700);
     opacity: 0;
-    z-index: 1;
```

Comment:
If you remove that one, the thumb ripple on hover and focus will be displayed above the thumb-label.
<img width="75" alt="Screen Shot 2022-01-24 at 11 10 44 AM" src="https://user-images.githubusercontent.com/59446295/150754248-332dadef-b5e8-4bca-b16b-187b9cf96101.png">
<img width="77" alt="Screen Shot 2022-01-24 at 11 09 20 AM" src="https://user-images.githubusercontent.com/59446295/150754274-7725dc69-868e-406c-969a-43e8ce215c42.png">

Reply:
I have swapped the elements in the template so that they appear correctly.